www/js/diary/LabelDetailsScreen.tsx

```diff
@@ -12,18 +12,19 @@ import { useTranslation } from "react-i18next";
 import MultilabelButtonGroup from "../survey/multilabel/MultiLabelButtonGroup";
 import UserInputButton from "../survey/enketo/UserInputButton";
 import { getAngularService } from "../angular-react-helper";
+import { useImperialConfig } from "../config/useImperialConfig";
 
 const LabelScreenDetails = ({ route, navigation }) => {
 
   const { surveyOpt } = useContext(LabelTabContext);
+  const { getFormattedDistance, distanceSuffix } = useImperialConfig();
   const { t } = useTranslation();
   const { height: windowHeight } = useWindowDimensions();
   const { colors } = useTheme();
   const { trip } = route.params;
 
-  const ImperialConfig = getAngularService('ImperialConfig');
   const DiaryHelper = getAngularService('DiaryHelper');
-  const sectionsFormatted = DiaryHelper.getFormattedSectionProperties(trip, ImperialConfig);
+  const sectionsFormatted = DiaryHelper.getFormattedSectionProperties(trip, {getFormattedDistance, distanceSuffix});
```

Comment:
That function, `getFormattedSectionProperties`, accepted ImperialConfig as a parameter because it used those two functions: `getFormattedDistance` and `distanceSuffix`.

This is a temporary way of providing those functions to DiaryHelper while both services are in the process of being migrated